t/t1070-issue749.t

```diff
@@ -0,0 +1,258 @@
+	flux queue start --all
+'
+
+test_expect_success 'add queue to association; update plugin' '
+	flux account edit-user user1 --queues=bronze &&
+	flux account-priority-update -p ${DB}
+'
+
+test_expect_success 'submit two jobs and make sure second one is held in DEPEND' '
+	job1=$(flux python ${SUBMIT_AS} 50001 \
+		-N 1 -n 1 --queue=bronze sleep inf) &&
+	flux job wait-event -vt 3 ${job1} alloc &&
+	job2=$(flux python ${SUBMIT_AS} 50001 \
+		-N 1 -n 1 --queue=bronze sleep inf) &&
+	flux job wait-event -vt 10 \
+		--match-context=description="max-run-jobs-queue" \
+		${job2} dependency-add &&
+	flux jobtap query mf_priority.so > query.json &&
+	test_debug "jq -S . <query.json" &&
+	jq -e \
+		".mf_priority_map[] |
+		 select(.userid == 50001) |
+		 .banks[0].held_jobs |
+		 length == 1" <query.json
+'
+
+test_expect_success 'update running jobs limit for queue; update plugin' '
+	flux account edit-queue bronze --max-running-jobs=-1 &&
+	flux account view-queue bronze > bronze_jobs_limits.out &&
+	grep "\max_running_jobs\": 100" bronze_jobs_limits.out &&
+	flux account-priority-update -p ${DB}
+'
+
+test_expect_success 'ensure previously held job proceeds to RUN' '
+	flux job wait-event -vt 5 \
+		--match-context=description="max-run-jobs-queue" \
+		${job2} dependency-remove
+'
+
+test_expect_success 'cancel job(s)' '
+	flux cancel ${job1} &&
+	flux cancel ${job2}
+'
+
+# This set repeats the same process as above, but ensures that a
+# max-nodes-per-association per-queue limit can also be removed from a job in
+# DEPEND state after an update.
+test_expect_success 'edit the max_nodes property for bronze queue; update plugin' '
+	flux account edit-queue bronze --max-nodes-per-assoc=1 &&
+	flux account-priority-update -p ${DB}
+'
+
+test_expect_success 'submit two jobs and make sure second one is held in DEPEND' '
+	job1=$(flux python ${SUBMIT_AS} 50001 \
+		-N 1 -n 1 --queue=bronze sleep inf) &&
+	flux job wait-event -vt 3 ${job1} alloc &&
+	job2=$(flux python ${SUBMIT_AS} 50001 \
+		-N 1 -n 1 --queue=bronze sleep inf) &&
+	flux job wait-event -vt 10 \
+		--match-context=description="max-resources-queue" \
+		${job2} dependency-add &&
+	flux jobtap query mf_priority.so > query.json &&
+	test_debug "jq -S . <query.json" &&
+	jq -e \
+		".mf_priority_map[] |
+		 select(.userid == 50001) |
+		 .banks[0].held_jobs |
+		 length == 1" <query.json
+'
+
+test_expect_success 'update max nodes per-association limit for queue; update DB' '
+	flux account edit-queue bronze --max-nodes-per-assoc=-1 &&
+	flux account view-queue bronze > bronze_max_nodes_limit.out &&
+	grep "\"max_nodes_per_assoc\": \"unlimited\"" bronze_max_nodes_limit.out &&
+	flux account-priority-update -p ${DB}
+'
+
+test_expect_success 'ensure previously held job proceeds to RUN' '
+	flux job wait-event -vt 5 \
+		--match-context=description="max-resources-queue" \
+		${job2} dependency-remove
+'
+
+test_expect_success 'cancel job(s)' '
+	flux cancel ${job1} &&
+	flux cancel ${job2}
```

Comment:
No need to make a change here, but just fyi `flux cancel` can take multiple jobids on the command line:
```sh
    flux cancel ${job1} ${job2}
```